corehq/warehouse/transforms/sql/user_group_dim.sql

```diff
@@ -13,7 +14,8 @@ SELECT
     gd.id,
     now(),
     now(),
-    combined_user_group_table.deleted
+    combined_user_group_table.deleted,
+    '{{ batch_id }}'
```

Comment:
i know we've talked about some optimizations in regards to the location dim using the batch_id. i'm tracking that, but haven't implemented it in this PR